README.md

```diff
@@ -196,171 +196,12 @@ export function a11yAuditIf(contextSelector, axeOptions) {
 
 ### Development Usage
 
-Usage in development is restricted to applications using Ember 1.13 and up, as it
-relies on the `didRender` hook of a component's life-cycle (a feature only
-available in versions of Ember with the Glimmer rendering engine).
+While this addon previously included a number of components that would aid in identifying axe violations during development, those have been deprecated in favor of other, industry standard tools such as:
```

Comment:
Let's reword this to be focused on the go forward users, not about migration to 4.0.0. We should author a custom transition guide for migrating from 3.x to 4.x and we can list the concrete things for those users in that guide.

Reply:
Sounds good.